frame/staking/src/pallet/impls.rs

```diff
@@ -668,12 +668,9 @@ impl<T: Config> Pallet<T> {
 
 	/// Clear all era information for given era.
 	pub(crate) fn clear_era_information(era_index: EraIndex) {
-		#[allow(deprecated)]
-		<ErasStakers<T>>::remove_prefix(era_index, None);
-		#[allow(deprecated)]
-		<ErasStakersClipped<T>>::remove_prefix(era_index, None);
-		#[allow(deprecated)]
-		<ErasValidatorPrefs<T>>::remove_prefix(era_index, None);
+		let _ = <ErasStakers<T>>::clear_prefix(era_index, u32::MAX, None);
```

Comment:
hey @ggwpez , added debug assertion that checks if the returned cursor is None for each of the three clear_prefix calls. It can help ensure that the tests catch any unexpected behavior.